client/landing/stepper/hooks/use-send-email-verification.ts

```diff
@@ -1,7 +1,14 @@
 import wpcom from 'calypso/lib/wp';
 
-export function useSendEmailVerification() {
+export interface ResendEmailVerificationBody {
+	from: string; // Helps decide the url of the page after confirmation.
+}
+
+export function useSendEmailVerification( body: ResendEmailVerificationBody ) {
```

Comment:
Do we need to make this param optional for all the places calling this (or the resent equivalent) without a passed argument?